nautobot/users/tests/test_views.py

```diff
@@ -40,6 +43,37 @@ def test_change_password_enabled(self):
         )
         self.assertBodyContains(post_response, "The two password fields")
 
+    @load_event_broker_override_settings(
+        EVENT_BROKERS={
+            "SyslogEventBroker": {
+                "CLASS": "nautobot.core.events.SyslogEventBroker",
+                "TOPICS": {
+                    "INCLUDE": ["*"],
+                },
+            }
+        }
+    )
+    def test_change_password(self):
+        self.user.set_password("foo")
+        self.user.save()
+        self.client.force_login(self.user)
+        with self.assertLogs("nautobot.events") as cm:
+            self.client.post(
+                reverse("user:change_password"),
+                data={
+                    "old_password": "foo",
+                    "new_password1": "bar",
+                    "new_password2": "bar",
+                },
+            )
+        payload = {"data": serialize_object_v2(self.user)}
+        self.assertEqual(
+            cm.output,
+            [f"INFO:nautobot.events.nautobot.users.user.change_password:{json.dumps(payload, indent=4)}"],
+        )
```

Comment:
Similarly would be good to assert that neither `password` nor `foo` nor `bar` appear in the event payload.